src/language-js/comments.js

```diff
@@ -831,6 +844,29 @@ function handleVariableDeclaratorComments(
   return false;
 }
 
+function handleTSFunctionTrailingComments(
+  text,
+  enclosingNode,
+  followingNode,
+  comment,
+  options
+) {
+  if (
+    followingNode ||
+    !enclosingNode ||
+    enclosingNode.type !== "TSMethodSignature" ||
```

Comment:
We seem to need to do the same for `TSDeclareFunction`, `MethodDefinition` and `TSAbstractMethodDefinition`.

Reply:
ok, i'm going to look into this case to

Reply:
Ok, please write when this PR is ready to review.